### ISSUE TITLE
Update dependencies to current versions

### ISSUE THREAD

**Issue by kaelumania** [jarednorman/bugsnag-elixir#46]
Use newer `poison` version:

```
Failed to use "poison" (version 3.1.0) because
  bugsnag (version 1.4.0) requires ~> 1.5 or ~> 2.0
  exq (version 0.8.7) requires >= 1.2.0 or ~> 2.0
  phoenix (version 1.3.0-rc.1) requires ~> 2.2 or ~> 3.0
  mix.lock specifies 3.1.0

** (Mix) Hex dependency resolution failed, relax the version requirements of your dependencies or unlock them (by using mix deps.update or mix deps.unlock). If you are unable to resolve the conflicts you can try overriding with {:dependency, "~> 1.0", override: true}
```

**Comment by dnsbty** [comment]
This would be fixed by #45

**Comment by bomberby** [comment]
#45 was merged

**Comment by mootpointer** [comment]
Closing, because this should have been fixed by #45.

Feel free to reopen if you're still having issues.
